test/micromamba/test_shell.py

```diff
@@ -152,20 +158,22 @@ def custom_shell(shell):
             assert res["actions"]["print"][0] == ""
 
     @pytest.mark.parametrize("shell_type", ["bash", "posix", "powershell", "cmd.exe"])
-    @pytest.mark.parametrize("root", [False, True])
-    @pytest.mark.parametrize("env_exists", [False, True])
+    @pytest.mark.parametrize("prefix_is_root", [False, True])
+    @pytest.mark.parametrize("prefix_exists", [False, True])
     @pytest.mark.parametrize("expanded_home", [False, True])
     @pytest.mark.parametrize("prefix_type", ["prefix", "name"])
```

Comment:
expanded_home case can be combined with prefix_type
```suggestion
    @pytest.mark.parametrize("prefix_type", ["prefix", "prefix_expanded_home", "name"])
```
and then made into a separate condition in the `if prefix_type ...` block

Reply:
thanks for the suggestion, yes we can have a lighter test matrix here avoiding unnecessary tests